### DISCUSSION TITLE
ext install not working

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#505] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
uhh,,,
[Screen recording 2023-02-08 2.48.34 PM.webm](https://user-images.githubusercontent.com/117109200/217648001-cafd8913-1bea-4aea-9add-15d7d46adfc9.webm)

**Comment by 3kh0** [comment, marked as answer]
@Aka-unblock no clue

**Reply by obkduishinijf** [reply]
i mean.. can you give me what the code was so i can atleast try it?

**Reply by Aka-but-nice** [reply]
Fixed, the site should be up again.
